source/management/identities.yml

```diff
@@ -749,6 +749,8 @@ definitions:
         $ref: '../shared/terminators.yml#/definitions/terminatorCostMap'
       appData:
         $ref: '../shared/base-entity.yml#/definitions/tags'
+      permissions:
+        $ref: '../shared/base-entity.yml#/definitions/permissions'
```

Comment:
Ahhh, there you are, my sweet, sweet child. Welcome back to this cruel cruel world.